docs/reference/docs/index_.asciidoc

```diff
@@ -188,25 +198,33 @@ will automatically be set to `create`. Here is an example (note the
 
 [source,js]
 --------------------------------------------------
-$ curl -XPOST 'http://localhost:9200/twitter/tweet/' -d '{
+POST twitter/tweet/
+{
     "user" : "kimchy",
     "post_date" : "2009-11-15T14:12:12",
     "message" : "trying out Elasticsearch"
-}'
+}
 --------------------------------------------------
+// AUTOSENSE
 
 The result of the above index operation is:
 
 [source,js]
 --------------------------------------------------
 {
+    "_shards" : {
+        "total" : 2,
+        "failed" : 0,
+        "successful" : 2
+    },
     "_index" : "twitter",
     "_type" : "tweet",
     "_id" : "6a8ca01c-7896-48e9-81cc-9f70661fcb32",
     "_version" : 1,
     "created" : true
 }
 --------------------------------------------------
+// TESTRESPONSE[s/6a8ca01c-7896-48e9-81cc-9f70661fcb32/$body._id/ s/"successful" : 2/"successful" : 1/]
```

Comment:
I like this trick - it is a "thoughtful" use of `s///` rather than a lying one. It gives the user an example of a regular generated _id while still letting us test everything else.